app/helpers/format-percentage.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+import prettyFloat from 'code-corps-ember/utils/pretty-float';
+
+const {
+  Helper: { helper }
+} = Ember;
+
+function applyFormatting(percentage) {
+  return percentage.length == 0 ? '' : `${percentage}%`;
+}
+
+/**
+ * Helper formats a float or a string representing a float into a percentage value.
+ *
+ * By default, the value will be formatted as `'22.25%'`.
+ * If the number is an integer, the format will be `'22.00%'`
```

Comment:
I know this adds some complexity, but I would prefer the percentage to have only a single decimal place., so `22.3%` instead of `22.25%`. The latter is just too much detail.